<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>444</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>RSA šifra</string>
  </property>
  <widget class="QRadioButton" name="radioButtonSiforvat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Šifrovat</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOdsifrovat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Odšifrovat</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVstupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vstupní data</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVystupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Výstupní data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProved">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vypočítej</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNacist">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst text ze souboru</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUloz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit výstup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Klíčové páry</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditN">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVstup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVystup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditD">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditE">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Veřejné</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Privátní</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRON">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>999999999</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditROE">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditROD">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRON_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGEN">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generovat klíče</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonulozklice">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit klíče</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNactiklic">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst klíče</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
